velox/common/memory/Memory.h

```diff
@@ -67,87 +67,90 @@ constexpr uint16_t kDefaultAlignment = 64;
 
 class ScopedMemoryPool;
 
-class MemoryPool {
+class MemoryPool : public std::enable_shared_from_this<MemoryPool> {
```

Comment:
Would you add class-level documentation to explain MemoryPool's ownership model?

Reply:
Added. Thanks!